packages/react-native/src/error-tracking/index.ts

```diff
@@ -35,4 +51,70 @@ export class ErrorTracking {
       ...additionalProperties,
     } as unknown as PostHogEventProperties)
   }
+
+  private resolveAutocaptureOptions(autocapture: AutocaptureOptions | boolean = false): AutocaptureOptions {
+    if (typeof autocapture === 'boolean') {
+      return {
+        uncaughtExceptions: autocapture,
+        unhandledRejections: autocapture,
+      }
+    }
+    return {
+      uncaughtExceptions: !!autocapture.uncaughtExceptions,
+      unhandledRejections: !!autocapture.unhandledRejections,
+    }
+  }
+
+  private autocaptureUncaughtErrors() {
```

Comment:
this might change if its web/mobile/macos etc
https://github.com/getsentry/sentry-react-native/blob/0b647531c13884b6be72c6f652482d79c6d2d96f/packages/core/src/js/integrations/reactnativeerrorhandlers.ts#L57
we need to make sure we are not crashing on the other platforms if the signature or the handlers dont exist in certain platforms